src/components/text-editor/__internal__/text-editor.style.js

```diff
@@ -54,6 +54,8 @@ StyledEditorContainer.defaultProps = {
 const StyledEditorOutline = styled.div`
   ${({ theme, isFocused, hasError }) => css`
     outline: none;
+    min-width: 320px;
```

Comment:
Why is this needed for the tooltips?

Reply:
It isn't I noticed a style issue in `TextEditor` when the window resized so added a fix here as it hadn't been raised and was a one liner

Reply:
In retrospect might have been worth doing it separately as would have reduced the files by one at least 😆